08_4_softmax_pg_pong.py

```diff
@@ -53,7 +53,7 @@
 
 # Loss function
 # Sum (Ai*logp(yi|xi))
-log_lik = -Y * tf.log(action_pred)
+log_lik = -Y * (tf.log(action_pred)+0.0000000001)
```

Comment:
Sorry. I made a mistake like you said. It should be like this.

log_lik = -Y * (tf.log(tf.clip_by_value(action_pred, 1e-10 , 1.0)))